.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!-- Please put the pull request description above -->
```

Comment:
Add our standard copyright header?

Reply:
I mean, if the thing emphasises our copyright policy, then it should set a good example and have a header. :)
Unless that's technically impossible, but I suppose there must be some comment syntax that would hide it from being displayed?